invite/test/github.test.ts

```diff
@@ -0,0 +1,137 @@
+/**
+ * Copyright 2020 The AMP HTML Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {createTokenAuth} from '@octokit/auth';
+import nock from 'nock';
+import {Octokit} from '@octokit/rest';
+
+import {getFixture} from './fixtures';
+import {GitHub} from '../src/github';
+
+describe('GitHub interface', () => {
+  const githubClient: Octokit = new Octokit({
+    authStrategy: createTokenAuth,
+    auth: '_TOKEN_',
+  });
+  let github: GitHub;
+
+  beforeAll(() => {
+    nock.disableNetConnect();
+  });
+
+  afterAll(() => {
+    nock.enableNetConnect();
+  });
+
+  beforeEach(() => {
+    github = new GitHub(githubClient, 'test_org');
+  });
+
+  afterEach(() => {
+    // Fail the test if there were unused nocks.
+    if(!nock.isDone()) {
+      throw new Error('Not all nock interceptors were used!');
+      nock.cleanAll();
+    }
+  });
+
+  describe('userIsMember', () => {
```

Comment:
Change the tests so that they'll expect the correct value for each case, then change the `it` to `it.skip`. Better have the real test at the ready

Reply:
These tests already are expecting the correct value (they currently fail). Added the `.skip` to the top `describe`.